tests/integrations/coprocessor/test_checksum.rs

```diff
@@ -7,7 +7,8 @@ use crc::crc64::{self, Digest, Hasher64};
 use kvproto::coprocessor::{KeyRange, Request};
 use kvproto::kvrpcpb::{Context, IsolationLevel};
 use protobuf::Message;
-use tipb::checksum::{ChecksumAlgorithm, ChecksumRequest, ChecksumResponse, ChecksumScanOn};
+
```

Comment:
Why add an empty line?

Reply:
no reason, probably the result of combining multiple patches and rebases. I can remove it.